### DISCUSSION TITLE
unable to view 3D brain model

### DISCUSSION THREAD

**Discussion by Wufoofoo** [neurodesk/neurodesk.github.io-hugo#379] [Q&A] [answered]
Hi, I was trying to view the brain in 3D but I was unable to see the 3D image using MRview in Neurodesk. Nothing appeared and it was just all black no matter how I drag it or change the threshold. Is there a way to fix this?

**Comment by stebo85** [comment]
Which image were you trying to visualise? Which version of mrtrix did you use?


Thank you

Steffen

--

W: https://mri.sbollmann.net<https://mri.sbollmann.net/>  | W: https://www.neurodesk.org<https://www.neurodesk.org/>  | T: https://twitter.com/sbollmann_MRI  | G: https://github.com/stebo85

Book meeting: https://calendly.com/s-bollmann/meeting

________________________________
From: Wufoofoo ***@***.***>
Sent: Monday, November 6, 2023 3:41:42 PM
To: NeuroDesk/neurodesk.github.io ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Neurodesk Organisation] unable to view 3D brain model (Discussion #379)


Hi, I was trying to view the brain in 3D but I was unable to see the 3D image using MRview in Neurodesk. Nothing appeared and it was just all black no matter how I drag it or change the threshold. Is there a way to fix this?

**Reply by Wufoofoo** [reply]
Hi Steffen, I used .mgz image file to visualise T1 brain MRI image in 3D. The version of mrtrix is 3.3.0.

**Reply by stebo85** [reply, marked as answer]
Dear @Wufoofoo - the images I tested it with work:

<img width="979" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/c130b477-7bfb-42eb-b5e4-8e5593de0e47">

A couple of things to try for you:
1) Can you open a standard image from /opt/fsl-6.0.3/data/standard/avg152T1.nii.gz -> does this work for you?
2) Can you share the image that doesn't work for you here?
3) Can you visualize the mgz image in freesurfers freeview?
